src/Bitcoin/TransactionSigner.h

```diff
@@ -11,11 +11,11 @@
 #include "Transaction.h"
 #include "TransactionBuilder.h"
 #include "TransactionInput.h"
+#include "../Groestlcoin/Transaction.h"
 #include "../Hash.h"
 #include "../PrivateKey.h"
 #include "../Result.h"
 #include "../Zcash/Transaction.h"
-#include "../Groestlcoin/Transaction.h"
```

Comment:
what is `Groestlcoin/Transaction ` import doing here? is it expected?

Reply:
Yes, I re-format the code